### PR TITLE
 Adding 'force_disconnect()' method

### PR DESCRIPTION
**Pull request by normanargueta**

### CODE REVIEW

**Comment by jenstroeger** [comment]
Happy to look through the code later, but at first glance I see a few issues that [pylint](https://www.pylint.org/) will catch.

File: opentok/opentok.py [hunk 1]
```diff
@@ -534,6 +535,31 @@ def get_stream(self, session_id, stream_id):
...
+        client will be disconnected
+
+        :param String connection_id: The connection ID of the client that will be disconnected
+        """
+        endpoint = self.endpoints.force_disconnect_url(session_id, connection_id)
+        response = requests.delete(
+            endpoint, headers=self.json_headers(), proxies=self.proxies, timeout=self.timeout
+        )
+
+        if response.status_code < 300:
```
Comment (msach22) [inline opentok/opentok.py]: I think we should specify this to `204`

**Comment by msach22** [comment]
@jenstroeger Great idea - I've created an issue for the linter: https://github.com/opentok/Opentok-Python-SDK/issues/134

But I think we should add that as a separate PR because we should add it for the whole lib

**Comment by jenstroeger** [comment]
> LGTM

@msach22, uhm… ?

**Pull request merged**